### PR TITLE
New resource `duplocloud_aws_apigateway_event` added.

### PR DESCRIPTION
**Pull request by tahir-duplo**
## Overview

New resource `duplocloud_aws_apigateway_event` added.

## Summary of changes

This PR does the following:

- ...
- ...

## Testing performed

- [ ] Using unit tests
- [x] Manually, on my local system
- [ ] Manually, on a remote test system

## Describe any breaking changes

- ...

### CODE REVIEW

File: docs/resources/aws_apigateway_event.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+resource "duplocloud_aws_apigateway_event" "apigateway_event" {
+  tenant_id      = duplocloud_tenant.myapp.tenant_id
+  api_gateway_id = "t84tb3skz0"
+  method         = "POST"
+  path           = "/v2/docs"
+  cors           = true
+  #   authorization_type = "CUSTOM"
+  #   authorizer_id      = "twiws3"
```
Comment (Duplo-nikv) [inline docs/resources/aws_apigateway_event.md:28]: remove commented line if not used in example

File: examples/resources/duplocloud_aws_apigateway_event/resource.tf [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+resource "duplocloud_aws_apigateway_event" "apigateway_event" {
+  tenant_id      = duplocloud_tenant.myapp.tenant_id
+  api_gateway_id = "t84tb3skz0"
+  method         = "POST"
+  path           = "/v2/docs"
+  cors           = true
+  #   authorization_type = "CUSTOM"
+  #   authorizer_id      = "twiws3"
```
Comment (Duplo-nikv) [inline examples/resources/duplocloud_aws_apigateway_event/resource.tf:13]: remove commented line if not used

**Review by Duplo-nikv — COMMENTED** [review]
Remove commented line in example if not needed

**Review by Duplo-nikv — APPROVED** [review]
Seems good

**Review by ddiaz-duplo — APPROVED** [review]

**Pull request merged**